Traceback (most recent call last):
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/home/mkzia/anaconda3/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/mkzia/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import seaborn as sns
sns.set(color_codes=True)
sns.regplot(x=x, y=y);
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/tmp/ipykernel_12373/1637144644.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mimport[0m [0mseaborn[0m [0;32mas[0m [0msns[0m[0;34m[0m[0;34m[0m[0m
[1;32m      2[0m [0msns[0m[0;34m.[0m[0mset[0m[0;34m([0m[0mcolor_codes[0m[0;34m=[0m[0;32mTrue[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 3[0;31m [0msns[0m[0;34m.[0m[0mregplot[0m[0;34m([0m[0mx[0m[0;34m=[0m[0mx[0m[0;34m,[0m [0my[0m[0;34m=[0m[0my[0m[0;34m)[0m[0;34m;[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m~/anaconda3/lib/python3.9/site-packages/seaborn/_decorators.py[0m in [0;36minner_f[0;34m(*args, **kwargs)[0m
[1;32m     44[0m             )
[1;32m     45[0m         [0mkwargs[0m[0;34m.[0m[0mupdate[0m[0;34m([0m[0;34m{[0m[0mk[0m[0;34m:[0m [0marg[0m [0;32mfor[0m [0mk[0m[0;34m,[0m [0marg[0m [0;32min[0m [0mzip[0m[0;34m([0m[0msig[0m[0;34m.[0m[0mparameters[0m[0;34m,[0m [0margs[0m[0;34m)[0m[0;34m}[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 46[0;31m         [0;32mreturn[0m [0mf[0m[0;34m([0m[0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     47[0m     [0;32mreturn[0m [0minner_f[0m[0;34m[0m[0;34m[0m[0m
[1;32m     48[0m [0;34m[0m[0m

[0;32m~/anaconda3/lib/python3.9/site-packages/seaborn/regression.py[0m in [0;36mregplot[0;34m(x, y, data, x_estimator, x_bins, x_ci, scatter, fit_reg, ci, n_boot, units, seed, order, logistic, lowess, robust, logx, x_partial, y_partial, truncate, dropna, x_jitter, y_jitter, label, color, marker, scatter_kws, line_kws, ax)[0m
[1;32m    849[0m ):
[1;32m    850[0m [0;34m[0m[0m
[0;32m--> 851[0;31m     plotter = _RegressionPlotter(x, y, data, x_estimator, x_bins, x_ci,
[0m[1;32m    852[0m                                  [0mscatter[0m[0;34m,[0m [0mfit_reg[0m[0;34m,[0m [0mci[0m[0;34m,[0m [0mn_boot[0m[0;34m,[0m [0munits[0m[0;34m,[0m [0mseed[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m    853[0m                                  [0morder[0m[0;34m,[0m [0mlogistic[0m[0;34m,[0m [0mlowess[0m[0;34m,[0m [0mrobust[0m[0;34m,[0m [0mlogx[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/anaconda3/lib/python3.9/site-packages/seaborn/regression.py[0m in [0;36m__init__[0;34m(self, x, y, data, x_estimator, x_bins, x_ci, scatter, fit_reg, ci, n_boot, units, seed, order, logistic, lowess, robust, logx, x_partial, y_partial, truncate, dropna, x_jitter, y_jitter, color, label)[0m
[1;32m    107[0m [0;34m[0m[0m
[1;32m    108[0m         [0;31m# Extract the data vals from the arguments or passed dataframe[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 109[0;31m         self.establish_variables(data, x=x, y=y, units=units,
[0m[1;32m    110[0m                                  x_partial=x_partial, y_partial=y_partial)
[1;32m    111[0m [0;34m[0m[0m

[0;32m~/anaconda3/lib/python3.9/site-packages/seaborn/regression.py[0m in [0;36mestablish_variables[0;34m(self, data, **kws)[0m
[1;32m     53[0m             [0;32mif[0m [0mnp[0m[0;34m.[0m[0mndim[0m[0;34m([0m[0mvector[0m[0;34m)[0m [0;34m>[0m [0;36m1[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m     54[0m                 [0merr[0m [0;34m=[0m [0;34m"regplot inputs must be 1d"[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 55[0;31m                 [0;32mraise[0m [0mValueError[0m[0;34m([0m[0merr[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     56[0m             [0msetattr[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mvar[0m[0;34m,[0m [0mvector[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     57[0m [0;34m[0m[0m

[0;31mValueError[0m: regplot inputs must be 1d
ValueError: regplot inputs must be 1d

